java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerController.java

```diff
@@ -18,4 +21,10 @@ public void createNewCustomer(){
         customerService.createNewCustomer();
     }
 
+    @GetMapping("/customersList")
+    public List<Customer> allCustomers(){
+        System.out.println("Getting ALL Customers");
+        return customerService.gettingAllCustomers();
+    }
+
```

Comment:
the url path is not Restful. This should just be `/customers`

Reply:
also, sysout or log?

Reply:
Done.